starter_code/src/com/joelimyx/MyLinkedList.java

```diff
@@ -13,14 +13,43 @@ public int getSize() {
 
 	public boolean remove(int index) {
 		//to-do
+		if (index > size){
+			return false;
+		}
+		Node<T> iterator = head;
+		for (int i = 0; i < index; i++) {
+			iterator = iterator.getNext();
+		}
+
+		Node<T> temp = iterator.getNext();
+		iterator.setNext(temp);
```

Comment:
This works, as long as `index` isn't zero. If it is zero (i.e. if the caller wants to remove the head), then I don't think your algorithm will do anything